src/main/java/com/microsoft/azure/relay/HybridConnectionListener.java

```diff
@@ -36,6 +37,7 @@
 	private URI address;
 	private TrackingContext trackingContext;
 	private TokenProvider tokenProvider;
+	private Throwable injectedFault;
```

Comment:
>injectedFault [](start = 19, length = 13)

Nit: If this is only ever used/throw during connect/re-connect could the name be updated to indicate that?

Reply:
For now that's the only possible one, I felt like it could be reused if there are more in the future

---
In reply to: [354522333](https://github.com/Azure/azure-relay-java/pull/58#discussion_r354522333) [](ancestors = 354522333)